CashSDK/src/main/java/cash/just/sdk/CashImpl.kt

```diff
@@ -123,11 +106,11 @@ class CashImpl:Cash {
         retrofit.register(sessionKey, phoneNumber, firstName, lastName).enqueue(object: Callback<WacBaseResponse> {
             override fun onResponse(call: Call<WacBaseResponse>, response: Response<WacBaseResponse>) {
                 if (response.isSuccessful) {
-                   if (response.body()?.result?.toLowerCase() == "ok") {
-                       listener.onSucceed()
-                   } else {
-                       listener.onError("error")
-                   }
+                    if (response.body()?.result?.toLowerCase() == "ok") {
```

Comment:
are you having a different formatter?

Reply:
I made another commit. Let me explain in a few words what I have done. The login method returns Response<WacBaseResponse>, It can not return Result<WacCallback> as you have described #39 here. Coroutines eliminate the need to set response results to the WacCallback. From your logic, a user has successfully logged in if the response is:  response.successful && response.body()?.result?.toLowerCase() == "ok" otherwise show the error msg.This is handled by AuthActivity now. Is it okay?